decode.go

```diff
@@ -102,7 +105,7 @@ func (de *decoder) message(buf []byte, sval reflect.Value) error {
 		rem, err := de.value(wiretype, buf, field)
 		if err != nil {
 			if fieldi < len(fields) && fields[fieldi] != nil {
-				return fmt.Errorf("Error while decding FieldName %s: %v", fields[fieldi].Name, err)
+				return fmt.Errorf("Error while decoding FieldName %s: %v", fields[fieldi].Name, err)
```

Comment:
HERE IT IS! I thought I fixed it, but didn't find it. I'll be happy to read `Error while decOding FieldName`. Can you make it a capital o? No, joke only...